docs/src/rules/accessor-pairs.md

```diff
@@ -17,15 +17,15 @@ Here are some examples:
 
 ```js
 // Bad
-var o = {
+let o = {
     set a(value) {
         this.val = value;
     }
 };
 
 
 // Good
-var o = {
+let o = {
```

Comment:
Can you change the variable names here and in all examples to avoid re-declaration?

`const` can be used where variables are not reassigned.